src/content/en/tools/chrome-devtools/_toc.yaml

```diff
@@ -49,12 +51,14 @@ toc:
         path: /web/tools/chrome-devtools/remote-debugging/webviews
     - title: Inspect and Debug JavaScript
       section:
+      - title: Overview
```

Comment:
Are you still planning on removing this section (as part of combining Overview into Get Started)?